src/sb_utils.R

```diff
@@ -29,4 +46,42 @@ sb_render_post_xml <- function(sb_id, ..., xml_file = NULL){
   
   sb_replace_files(sb_id = sb_id, xml_file)
   
-}
\ No newline at end of file
+}
+
+# Helper function to create a task_table for the files that need to be pushed to SB
+do_item_replace_tasks <- function(files, sb_id, task_yml, out_dir) {
+  # Define task table rows
+  tasks <- basename(files)
+  
+  # Define task table columns
+  sb_push <- scipiper::create_task_step(
+    step_name = 'push_file_to_sb',
+    target_name = function(task_name, step_name, ...){
+      sprintf("%s_pushed_to_sb", task_name)
+    },
+    command = function(task_name, ...){
+      sprintf("item_replace_files('%s', '%s')", sb_id, task_name)
+    } 
+  )
+  
+  # Create the task plan
+  task_plan <- create_task_plan(
+    task_names = tasks,
+    task_steps = list(sb_push),
+    final_steps = c('push_file_to_sb'),
+    add_complete = FALSE)
+  
+  # Create the task remakefile
+  final_target <- sprintf("%s/%s_files_uploaded.ind", out_dir, gsub(".yml", "", basename(task_yml)))
+  create_task_makefile(
+    task_plan = task_plan,
+    makefile = task_yml,
+    include = 'remake.yml',
```

Comment:
I don't think you need to `include` remake.yml. You can make this task table completely independent of the parent remake file as long as you pass in the `sb_id`

Ideally, I don't think you would specify the task_yml argument, and instead this function would create a task table with a generic name (or a name related to the `sb_id`) and then delete the yaml when the targets are complete, like I did here for cleanup: https://github.com/USGS-R/lake-temperature-model-prep/blob/f912d7d5143f6d279fad51cdb0a65ad704532b6d/6_wqp_fetch/src/fetch_wqp_data.R#L87

Reply:
I had it that way originally, but I wasn't sure if that was a good idea. Cool!